roles/tendrl-httpd-ssl/handlers/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+- name: restart httpd
+  service:
+    name=httpd
+    state=restarted
```

Comment:
Do we need full restart of `http` service? Wouldn't be enough `state: reloaded`?

Reply:
good point, fixed: 7bbe9c0c78743a19dd5e2c814947afc5e0cddbcb